build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
```

Comment:
@pweil- can you explain what you mean here? `build.sh` is following convention defined in https://docs.docker.com/engine/userguide/eng-image/multistage-build/#use-multi-stage-builds

are you suggesting we convert this build script into a go program?

+1 on adding a make target